packages/type-extrinsics/package.json

```diff
@@ -11,6 +11,6 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@babel/runtime": "^7.1.5",
-    "@polkadot/util": "^0.32.21"
+    "@polkadot/util": "^0.32.23"
```

Comment:
Shouldn't @polkadot/types be here? Idem in other package.jsons.

Reply:
Yeap. The same thing that got us in trouble the first time around. Will add to storage as well.

Reply:
And i think also in api, api-observable, rpc-core, rpc-rx etc